package.json

```diff
@@ -52,6 +52,7 @@
     "method-override": "3.0.0",
     "node-pushnotifications": "^1.0.20",
     "node-rest-client": "3.1.0",
+    "nyc": "false13.3.0",
```

Comment:
So I've noticed `yarn` likes to throw a "false" in front of version numbers. Do you know what this does? Do we need it?

Reply:
Good catch. Fixed.